test/auth_cluster/conftest.py

```diff
@@ -6,7 +6,7 @@
 # This file configures pytest for all tests in this directory, and also
 # defines common test fixtures for all of them to use
 from pytest import Config
-from test.topology.conftest import *
+from test.topology_custom.conftest import *
```

Comment:
Nitpick about the commit message: The "now that we support suite subfolder" statement is irrelevant here, as you do not - unless I missed something - actually introduce subdirectories in this patch.